constraint/pkg/client/drivers/local/builtin.go

```diff
@@ -20,7 +21,12 @@ func externalDataBuiltin(d *Driver) func(bctx rego.BuiltinContext, regorequest *
 			return externaldata.HandleError(http.StatusBadRequest, err)
 		}
 
-		externaldataResponse, statusCode, err := d.sendRequestToProvider(bctx.Context, &provider, regoReq.Keys)
+		clientCert, err := tls.LoadX509KeyPair(d.clientCertFile, d.clientKeyFile)
```

Comment:
Add tests for this functionality. As-is would likely be annoying, so you'll likely want to use `fs.FS` so that you can test this without needing to use actual files.

(Such tests would require switching to `tls.X509KeyPair` instead of `tls.LoadX509KeyPair`)